pkgs/applications/audio/faust/faust2.nix

```diff
@@ -221,7 +221,7 @@ let
 
       postFixup = ''
         for script in "$out"/bin/*; do
-          wrapProgram "$script" --prefix PATH : "${runtimePath}"
+          wrapProgram "$script" --suffix PATH : "${runtimePath}"
```

Comment:
I don't think we should change this. It seems like `runtimePath` _may_ include xdg-utils (I can't tell because it's just an function argument), but we shouldn't be suffixing any other dependencies.